scripts/theo/generate-example.ts

```diff
@@ -0,0 +1,29 @@
+import { writeFileSync } from 'fs';
+import { resolve } from 'path';
+import { convert, registerFormat, registerTransform } from 'theo';
+import { registerOdsValueTransform } from './helpers';
+
+const fileName = process.env.FILE_NAME;
+if (!fileName) throw new Error('No file name provided');
+
+const srcPath = resolve(__dirname, `src/${fileName}.json`);
+const formatPath = resolve(__dirname, 'example/html.cjs');
```

Comment:
It seems `html.cjs` is only imported in this script, and interpreted as an ES class.
You sure it can't be `.ts` and Theo is using the file directly?

Reply:
Yes, even if it's named as `.ts` it cannot be ES5 module.

Reply:
Ah, I see :/